examples/apps/echo/client/echo_client.py

```diff
@@ -126,22 +129,24 @@ def Main(args=None):
 
 	# Connect to registry list and retrieve registry
 	if not off_chain:
-		direct_wrapper.init_worker_registry_list(config)
-		registry_lookup_result = direct_wrapper.registry_lookup()
+		# Lookup returns tuple, first element is number of registries and
+		# Second element list of organization ids.
+		registry_lookup_result = direct_jrpc.create_worker_registry_list_adaptor(
+			config).registry_lookup()
 		if (registry_lookup_result[0] == 0):
 			logger.warn("No registries found")
 			sys.exit(1)
-		registry_retrieve_result = direct_wrapper.registry_retrieve(
-			registry_lookup_result[2][0])
+		# Retrieve the fist registry details.
+		registry_retrieve_result = direct_jrpc.create_worker_registry_list_adaptor(
+			config).registry_retrieve(registry_lookup_result[2][0])
```

Comment:
We should write a getter function to get registry_lookup_result[2][0] which takes "registry_lookup_result" as parameter. The client could call the getter function.

Reply:
worker_lookup will return tuple (number of workers matched the search criteria, lookup_tag, array of worker ids). It is better client to interpret it as it is mentioned in spec.

Reply:
The idea is to provide a function so that different clients can use this function to interpret the results consistently, We need to implement only 3 functions get_total_count(), get_lookupTag() and get_Ids()